content/docs/components-and-props.md

```diff
@@ -233,30 +259,40 @@ function Comment(props) {
 
 [](codepen://components-and-props/extracting-components-continued)
 
-Extracting components might seem like grunt work at first, but having a palette of reusable components pays off in larger apps. A good rule of thumb is that if a part of your UI is used several times (`Button`, `Panel`, `Avatar`), or is complex enough on its own (`App`, `FeedStory`, `Comment`), it is a good candidate to be a reusable component.
+Mengekstraksi komponen mungkin terlihat seperti pekerjaan yang kasar pada
+awalnya, namun mempunyai *pallete* dari komponen yang bisa digunakan kembali akan
+terbayar pada aplikasi yang lebih besar. Aturan praktik yang bagus adalah jika
+pada bagian dari UI digunakan beberapa kali (*Button*, *Panel*, *Avatar*), atau yang cukup
+kompleks (App, FeedStory, Comment), ini merupakan kandidat yang bagus untuk
+dijadikan sebagai komponen yang dapat digunakan kembali.
 
-## Props are Read-Only {#props-are-read-only}
+## Props adalah *Read-Only* {#props-are-read-only}
```

Comment:
Props bersifat Read-Only

What do you think? 😄

Reply:
Okay nice. Will fix all of this issue soon